pkg/recommender.py

```diff
@@ -425,3 +436,81 @@ def record_data(self, data):
 
     storing_db.close()
 
+  def _schedule_evt(self):
+    '''
+    Send the morning message at 10 am
+    '''
+    next_morning = ''
+    while True:
+      now = datetime.now()
+
+      #check if we have passed 10 am yet
+      if now.hour >= 10: #then you are currently in the day before
+        #add a day to the date
+        next_morning = now + timedelta(days =1)
+        #change the time to 10am
+        next_morning = next_morning.replace(hour = 10, minute = 0, second = 0, microsecond = 0)
```

Comment:
a trival format convention, when giving key parameter to a function, we dont need  space around assign operator (=). just `func(key=value)`

But yes, we like spaces in a statement